### ISSUE TITLE
placeholder check should be smarter

### ISSUE THREAD

**Issue by wojciechczerniak** [rossPatton/stylint#280]
`@extend` has more advanced syntax than currently supported. With [extending multiple selectors](http://stylus-lang.com/docs/extend.html#extending-multiple-selectors) and [optional extending](http://stylus-lang.com/docs/extend.html#optional-extending) the line can look like:

```
@extend .design !optional, $specialDesign !optional
```

and we should catch placeholder preferences not only at the beginning of the line.

Also should support multiple occurrences #266
```
@extend .firstPlaceholder, $firstPlaceholder, .anotherSelector, $anotherPlaceholder
```

**Comment by rossPatton** [comment]
it's a little smarter now, gonna close this and just reference #269 since that's the real issue here
